examples/Makefile

```diff
@@ -43,155 +35,117 @@ GATK_GERMLINE_FILES	=	gatk/GATK_Germline/data/ref/ref.dict \
-	pachctl create pipeline -f opencv/montage.json
-	pachctl put file images@master -i opencv/images.txt
-	pachctl put file images@master -i opencv/images2.txt
-
-opencv-testing: 
-	pachctl create repo images
-	$(JQ) "$(STATS)" opencv/edges.json | pachctl create pipeline -f -
-	$(JQ) "$(STATS)" opencv/montage.json | pachctl create pipeline -f -
-	pachctl start transaction
-	pachctl start commit images@master
-	pachctl finish transaction
-	pachctl put file images@master -i opencv/images.txt
-	pachctl put file images@master -i opencv/images2.txt
-	pachctl put file images@master -i opencv/images3.txt
-	pachctl put file images@master -i opencv/images4.txt
-	pachctl delete file images@master:/w7RVTsv.jpg
-	pachctl finish commit images@master
-
-opencv-testing-delete:
-	-pachctl delete pipeline montage
-	-pachctl delete pipeline edges
-
-opencv-delete: opencv-testing-delete
-	-pachctl delete repo images
-
-ml/object-detection/frozen_inference_graph.pb:
-	wget -p http://download.tensorflow.org/models/object_detection/ssd_mobilenet_v1_coco_11_06_2017.tar.gz --directory-prefix /tmp
-	tar -C ml/object-detection --strip-components 1 -xvf /tmp/download.tensorflow.org/models/object_detection/ssd_mobilenet_v1_coco_11_06_2017.tar.gz ssd_mobilenet_v1_coco_11_06_2017/frozen_inference_graph.pb
+opencv-base:
+	$(PACHCTL) create repo images
+	$(PACHCTL) create pipeline -f opencv/edges.json
+	$(PACHCTL) create pipeline -f opencv/montage.json
 
+opencv: opencv-base
+	$(PACHCTL) put file images@master -i opencv/images.txt
+	$(PACHCTL) put file images@master -i opencv/images2.txt
 
-object-detection-images-repo: ml/object-detection/frozen_inference_graph.pb
-	pachctl create repo images
+opencv-delete:
+	$(PACHCTL) delete pipeline montage
+	$(PACHCTL) delete pipeline edges
+	$(PACHCTL) delete repo images
 
-object-detection-no-images-repo: ml/object-detection/frozen_inference_graph.pb
-
-object-detection-demo: object-detection-no-images-repo
-	pachctl create repo training
-	pachctl put file training@master:frozen_inference_graph.pb -f ml/object-detection/frozen_inference_graph.pb
-	pachctl put file images@master:dogs.jpg -f ml/object-detection/images/dogs.jpg
+ml/object-detection/frozen_inference_graph.pb:
+	$(WGET) -p http://download.tensorflow.org/models/object_detection/ssd_mobilenet_v1_coco_11_06_2017.tar.gz --directory-prefix /tmp
+	$(TAR) -C ml/object-detection --strip-components 1 -xvf /tmp/download.tensorflow.org/models/object_detection/ssd_mobilenet_v1_coco_11_06_2017.tar.gz ssd_mobilenet_v1_coco_11_06_2017/frozen_inference_graph.pb
 
-object-detection:  object-detection-images-repo object-detection-demo
-	pachctl create pipeline -f ml/object-detection/model.json
-	pachctl create pipeline -f ml/object-detection/detect.json
+OBJECT_MODEL_PIPELINE = pachctl create pipeline -f ml/object-detection/model.json
+OBJECT_DETECT_PIPELINE = pachctl create pipeline -f ml/object-detection/detect.json
+object-detection-base: 
+	@$(OBJECT_IMAGES)
```

Comment:
AFAICS there's no default set for `OBJECT_IMAGES`, so this won't work unless the variable is set

Reply:
Correct, it will be an empty operation, as intended.  The base- targets are intended to be built upon; I'm not sure how to make them not callable from outside.  That's currently beyond my make-fu. 

I'll make the fact that it's intended to be set by targets that *use* this one clearer.

Reply:
I've added the comments above.